build/charts/theia/values.yaml

```
@@ -297,3 +299,37 @@ theiaManager:
     tlsMinVersion: ""
   # -- Log verbosity switch for Theia Manager.
   logVerbosity: 0
+zookeeper:
+  storage:
+    # -- Zookeeper storage size. Can be a plain integer or as a fixed-point
+    # number using one of these quantity suffixes: E, P, T, G, M, K. Or the
+    # power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+    size: "1Gi"
+    createPersistentVolume:
+      # -- Type of PersistentVolume. Can be set to "Local" or "NFS". Please set
+      # this value to use a PersistentVolume created by Theia.
+      type: ""
+      local:
+        # -- The local path. Required when type is "Local".
+        path: "/data/zookeeper"
+        # -- Affinity for the Local PersistentVolume. By default it requires to
+        # label the Node used to store the Zookeeper data with
+        # "antrea.io/zookeeper-data-node=".
+        affinity: { }
```

Maybe remove the space here to keep consistent with previous empty object?
```suggestion
        affinity: {}
```

---

Fixed.